.editorconfig

```diff
@@ -11,3 +11,7 @@ insert_final_newline = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.yml]
+indent_style = space
```

Comment:
The indent style in WeCount is to use "tab" as it provides flexibility to map to desired number of spaces. Do you think if yml files can use it too?

Reply:
@cindyli tabs are not valid in YAML files. https://yaml.org/faq.html